### DISCUSSION TITLE
有没有办法设置成不翻译代码块？GitHub的代码块都被翻译了。

### DISCUSSION THREAD

**Discussion by bingoku** [immersive-translate/immersive-translate#1135] [Ideas]
### 新功能 | New Feature

设置所有的代码块，不允许被翻译。

### 为什么您想要添加这些功能 | Why you want to add these features

代码块整个是没有必要翻译的，即便里面的注释需要翻译，自己想办法。。。

**Comment by theowenyoung** [comment]
目前默认就是不翻译的，请问有 出现问题 的网址吗？

**Reply by bingoku** [reply]
比如阅读readme的时候 ：https://github.com/macosui/macos_ui
如果开启了，鼠标指向翻译，鼠标指到了，就会自动翻译代码。如果能优化一下就好了。
